.github/workflows/release-embedding-sdk.yml

```diff
@@ -249,7 +248,12 @@ jobs:
         run: |
           cd sdk
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_RELEASE_TOKEN }} > .npmrc
-          npm publish
+          npm publish --tag ${{fromJson('{"master": "canary", "release-x.51.x": "50-stable"}')[inputs.branch]}}
+
+      - name: Add `latest` tag to `release-x.51.x` branch deployment
+        if: ${{ inputs.branch == 'release-x.51.x' }}
+        run: |
+          npm dist-tag add @metabase/embedding-sdk-react@${{ env.sdk_version }} latest
```

Comment:
We can't publish an npm version with more than 1 dist-tags, so we need to add a new tag after the package has already been published.